core/src/main/java/org/elasticsearch/common/bytes/BytesArray.java

```diff
@@ -91,4 +121,11 @@ public long ramBytesUsed() {
         return bytes.length;
     }
 
+    @Override
+    public void close() {
+        // TODO: Do we want to throw an exception to expose misuse?
```

Comment:
the closeable contract says it's closeable multple times so we are good I guess.